vmb/matrimony/forms.py

```diff
@@ -153,6 +153,10 @@ def __init__(self, *args, **kwargs):
 
 
 class MatrimonyProfileBasicDetailsForm(BaseMatrimonyProfileForm):
+    actual_dob = forms.BooleanField(
```

Comment:
This field should be named `is_actual_dob`. What's the `default` value for this field? How will we handle migration for existing rows?